### PR TITLE
Add showErrorMessage store action

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Part of https://github.com/owncloud/web/issues/9449

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-app-files/tests/unit/composables/actions/files/useFileActionsCreateNewFolder.spec.ts [hunk 1]
```diff
@@ -58,13 +58,7 @@ describe('useFileActionsCreateNewFolder', () => {
...
           await addNewFolder('myfolder')
           await nextTick()
-          expect(storeOptions.actions.showMessage).toHaveBeenCalledWith(
-            expect.anything(),
-            expect.objectContaining({
-              title: 'Failed to create folder',
-              status: 'danger'
-            })
-          )
+          expect(storeOptions.actions.showErrorMessage).toHaveBeenCalled()
```
Comment (dschmidt) [inline packages/web-app-files/tests/unit/composables/actions/files/useFileActionsCreateNewFolder.spec.ts:61]: same here

File: packages/web-runtime/src/store/app.ts [hunk 2]
```diff
@@ -4,6 +4,12 @@ const state = {
 }
 
 const actions = {
+  showErrorMessage({ commit }, message) {
+    message.status = message.status || 'danger'
+    message.timeout = message.timeout || 0
+
+    commit('ENQUEUE_MESSAGE', message)
+  },
```
[the remark this answers is not in the document]
Comment (dschmidt) [reply packages/web-runtime/src/store/app.ts:12]: i.e. when you pass `desc` error object you extract it from that?

**Review by dschmidt — APPROVED** [review]

File: packages/web-runtime/src/store/app.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (JanAckermann) [reply packages/web-runtime/src/store/app.ts:12]: For example, but this is mostly reversed to the subtitle of notification. I would add a new property, like error

File: packages/web-app-files/tests/unit/composables/actions/files/useFileActionsCreateNewFolder.spec.ts (hunk 1 above)
Comment (JanAckermann) [reply packages/web-app-files/tests/unit/composables/actions/files/useFileActionsCreateNewFolder.spec.ts:61]: reintroduced

File: packages/web-app-files/tests/unit/composables/actions/files/useFileActionsCreateNewFile.spec.ts [hunk 3]
```diff
@@ -60,13 +60,7 @@ describe('useFileActionsCreateNewFile', () => {
...
           await addNewFile('myfolder', null)
           await nextTick()
-          expect(storeOptions.actions.showMessage).toHaveBeenCalledWith(
-            expect.anything(),
-            expect.objectContaining({
-              title: 'Failed to create file',
-              status: 'danger'
-            })
-          )
+          expect(storeOptions.actions.showErrorMessage).toHaveBeenCalled()
```
[the remark this answers is not in the document]
Comment (JanAckermann) [reply packages/web-app-files/tests/unit/composables/actions/files/useFileActionsCreateNewFile.spec.ts:63]: reintroduced

**Pull request merged**